src/App.tsx

```diff
@@ -3,36 +3,40 @@ import ActivitiesList from "@components/ActivitiesList/ActivitiesList";
 import { ACTIVITIES_MOCK } from "@components/ActivitiesList/mocks";
 import DataContainer from "@components/DataContainer/DataContainer";
 import HelloWorld from "@components/HelloWorld/HelloWorld";
-import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { BrowserRouter, Link, Route, Routes } from "react-router-dom";
 
 import styles from "./App.module.css";
 
 export default function App() {
   return (
-    <main className={styles.main}>
-      <img className={styles.logo} alt="React logo" width="400px" src={Logo} />
```

Comment:
Good idea to remove the React logo image at this point! Please also remove the file itself from the `/assets` folder

Reply:
my bad, this was actually just moved